Package.swift

```diff
@@ -25,10 +25,12 @@ let package = Package(
     dependencies: [
         .package(url: "https://github.com/apple/swift-log.git", from: "1.4.0"),
         .package(url: "https://github.com/apple/swift-metrics.git", "1.0.0"..<"3.0.0"),
+        .package(url: "https://github.com/apple/swift-distributed-tracing.git", from: "0.1.2"),
         .package(url: "https://github.com/apple/swift-nio.git", .upToNextMajor(from: "2.16.1")),
         .package(url: "https://github.com/apple/swift-nio-ssl.git", .upToNextMajor(from: "2.7.2")),
         .package(url: "https://github.com/apple/swift-nio-transport-services.git", .upToNextMajor(from: "1.0.0")),
-        .package(url: "https://github.com/swift-server/async-http-client.git", .upToNextMajor(from: "1.3.0")),
+//        .package(url: "https://github.com/swift-server/async-http-client.git", .upToNextMajor(from: "1.3.0")),
+        .package(url: "https://github.com/slashmo/async-http-client.git", .branch("feature/tracing")),
```

Comment:
would be great to have tracing in AHC, a lot of libraries depend on that

Reply:
Yep, that’s only a matter of time. Though for libraries it doesn’t make sense to merge in support yet because they would need to accept a `LoggingContext` in their APIs until Baggage will be passed through a task-local (Swift 5.5).